scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -86,14 +84,17 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
 
                 try
                 {
-                    // Iterate over every blob in the latest version folder and copy it to the 'latest' folder
-                    foreach (string blob in _latestBlobs)
-                    {
-                        string targetUrl = $"{targetContainer}{Path.GetFileName(blob)}"
-                                        .Replace(CliNuGetVersion, "latest")
-                                        .Replace(SharedFrameworkNugetVersion, "latest");
-                        AzurePublisherTool.CopyBlob(blob.Replace("/dotnet/", ""), targetUrl);
-                    }
+                    // Copy the latest CLI bits
+                    CopyBlobs($"{Channel}/Binaries/{CliNuGetVersion}/", targetContainer);
+                    
+                    // Copy the latest installer files
+                    CopyBlobs($"{Channel}/Installers/{CliNuGetVersion}/", $"{Channel}/Installers/Latest/");
+
+                    // Copy the shared framework installers
+                    CopyBlobs($"{Channel}/Installers/{SharedFrameworkNugetVersion}/", $"{Channel}/Installers/Latest/");
+
+                    // Copy the shared framework
+                    CopyBlobs($"{Channel}/Binaries/{SharedFrameworkNugetVersion}/", targetContainer);
 
```

Comment:
*nit* let's be consistent with calling convention here. I think we can do away with 'targetContainer'. Not this PR, tho.